src/abstractdataframe/abstractdataframe.jl

```diff
@@ -371,31 +371,36 @@ end
 # summarize the columns of a df
 # TODO: clever layout in rows
 """
-Summarize the columns of an AbstractDataFrame
-
 ```julia
-describe(df::AbstractDataFrame)
-describe(io, df::AbstractDataFrame)
+describe(df::AbstractDataFrame; stats = [:mean, :min, :median, :max, :Nmissing, :datatype])
+describe(io, df::AbstractDataFrame; stats = [:mean, :min, :median, :max, :Nmissing, :datatype])
 ```
 
 **Arguments**
 
 * `df` : the AbstractDataFrame
 * `io` : optional output descriptor
+* `stats::AbstractVector{Symbol}`: the summary statistics to report
 
 **Result**
 
-* nothing
+* A `DataFrame` where each row represents a variable and each column a summary statistic.
+
 
 **Details**
 
-If the column's base type derives from Number, compute the minimum, first
-quantile, median, mean, third quantile, and maximum. Missings are filtered and
-reported separately.
+If the column's base type derives from Number, compute the mean, standard 
+deviation, minimum, first quantile, median, third quantile, and maximum. If 
+a column is not numeric, these statistics are populated with `nothing`s. 
 
-For boolean columns, report trues, falses, and missings.
```

Comment:
Added, however this would change the behavior, as I mentioned in another comment, it doesn't currently get unique values for `Number` types, but maybe it should.